plugins/de.cau.cs.kieler.klighd/src/de/cau/cs/kieler/klighd/syntheses/DiagramLayoutOptions.java

```diff
@@ -201,7 +201,7 @@ private DiagramLayoutOptions() {
     /**
      * @see CoreOptions#PORT_LABEL_PLACEMENT
      */
-    public static final IProperty<PortLabelPlacement> PORT_LABEL_PLACEMENT =
+    public static final IProperty<EnumSet<PortLabelPlacement>> PORT_LABEL_PLACEMENT =
```

Comment:
That's an API breaking change, right? Did that change happen in ELK, too?

Reply:
Has the corresponding change in ELK been released?

Reply:
Yes, this was released in ELK 0.7.0